### PR TITLE
Protect: Fix recommendation when plugin doesn't have a fix yet

### PR DESCRIPTION
**Pull request by miguelxpn**
#### Changes proposed in this Pull Request:

* This hides the how to fix? text in cases where the plugin fixedIn is null.

#### Other information:
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Install this branch on your site.
* Install a plugin with a vulnerability that doesn't have a fix yet. (You can use this [vulnerability](https://wpscan.com/plugin/dx-share-selection) and rename one of your plugins to it as this particular plugin is closed)
* Verify that the How to fix? recommendation to update to null is not there anymore.

### CODE REVIEW

File: projects/plugins/protect/src/js/components/vulnerabilities-list/list.jsx [hunk 1]
```diff
@@ -20,15 +20,20 @@ const VulAccordionItem = ( { id, name, version, title, icon, fixedIn, type } ) =
...
-					/* translators: Translates to Update to. %1$s: Name. %2$s: Fixed version */
-					sprintf( __( 'Update to %1$s %2$s', 'jetpack-protect' ), name, fixedIn )
-				}
-			</Text>
+			{ fixedIn && (
+				<Text variant="title-small" mb={ 2 }>
+					{ __( 'How to fix it?', 'jetpack-protect' ) }
+				</Text>
+			) }
+			{ fixedIn && (
```
Comment (nateweller) [inline projects/plugins/protect/src/js/components/vulnerabilities-list/list.jsx]: Very minor, is there a reason for having two separate `fixedIn` checks vs grouping the title and message in a single check? i.e.
```javascript
{ fixedIn && (
    <>
        <Text variant="title-small">...</Text>
        <Text mb={ 2 }>...</Text>
    </>
) }
```

**Review by nateweller — DISMISSED** [review]
Looks good to me!

File: projects/plugins/protect/src/js/components/vulnerabilities-list/list.jsx (hunk 1 above)
Comment (miguelxpn) [reply projects/plugins/protect/src/js/components/vulnerabilities-list/list.jsx]: Nope, the IDE was complaining when I tried to group them and I didn't realize that I could just add an empty jsx element to be the parent of both and that would fix it. Thanks for the tip!

**Review by nateweller — APPROVED** [review]
Nice!

**Pull request merged**